go.test.sh

```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
 set -e
-echo "" > coverage.txt
+#set -x # debug/verbose execution
+
+# Ensures removal of file
+trap 'rm -f profile.out downloader/*.{mp4,download,jpg}' SIGINT EXIT
+
+# go -race needs CGO_ENABLED to proceed
+export CGO_ENABLED=1;
 
 for d in $(go list ./... | grep -v vendor); do
-    go test -race -coverprofile=profile.out -covermode=atomic "$d"
-    if [ -f profile.out ]; then
-        cat profile.out >> coverage.txt
-        rm profile.out
-    fi
+	go test -v -timeout=2m -race -coverprofile=profile.out -covermode=atomic "${d}"
```

Comment:
Remove this timeout, some tests take a long time, others LGTM

Reply:
they could, altho i have not seen any run past this time and 2 minutes is... well, a very very forgiven timeout. If a test has not finished after 2 minutes... well, maybe 56kb isn't the right connection speed for your needs.

You are more than welcome to change it - or heck, i could if i find time